application/src/main/java/com/github/pieter_groenendijk/model/ProductCopy.java

```diff
@@ -0,0 +1,54 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+
+@Entity
+@Table (name = "ProductCopy")
+public class ProductCopy {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long productCopyId;
+
+   @ManyToOne
+    @JoinColumn(name = "productId", nullable = false)
+    private PhysicalProduct physicalProduct;
+
+    @Column (name = "availabilityStatus", nullable = false, length = 50)
+    private String availabilityStatus;
+
+    @Column (name = "isDamaged", nullable = false)
+    private boolean isDamaged;
```

Comment:
Staat het hele idee van schade en het behandelen daarvan eigenlijk in de requirements? Is dat besproken met Ronald? Ik weet het oprecht niet.